addons/planner_project/static/src/less/planner_project.less

```diff
@@ -0,0 +1,62 @@
+
+#PlannerModal {
+
+    font-family: "Lato", serif;
+
+    #planner_page1 {
+        h1 {margin-bottom: 20px;}
+        mb32 {
+            margin-bottom: 32px;
+        }
+        mb8 {
+            margin-bottom: 8px;
+        }
+    }
+    #planner_page2,#planner_page3 {
+        textarea{
+            width: 100%;
+            max-width: 100%;
+        }
+    }
+    #planner_page4, #planner_page7, #planner_page8, #planner_page11, #planner_page14 {
+        .container-fluid {
+            .col-xs-12 {
+                padding: 5px;
+                &:first-child {
+                    padding-left: 0;
+                }
+                &:last-child {
+                    padding-right: 0;
+                }
+            }
+        }
+    }
+    #planner_page4, #planner_page10 {
+        li {
+            font-size: 14px;
+        }
+        select {
+            font-size: 14px;
+        }
+    }
+    #planner_page11 {
```

Comment:
not `inputbox` on that page, and we can survive without the `padding: 5px` so you can remove it